srt-protocol/src/options/connect.rs

```diff
@@ -23,6 +23,35 @@ pub struct Connect {
     ///
     /// The default value is 0x010000 (SRT v1.0.0).
     pub min_version: SrtVersion,
+
+    /// SRTO_UDP_RCVBUF
+    ///
+    /// UDP Socket Receive Buffer Size. Configured in bytes, maintained in packets based on MSS value.
+    /// Receive buffer must not be greater than FC size.
```

Comment:
You should add validation for this, and I'm guessing the send buffer ought to have the same limit enforced too.